src/styles/index.scss

```
@@ -1,3 +1,154 @@
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #f4f4f4;
+}
+
+.product {
+  display: flex;
+  justify-content: center;
+
+  &-image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &-card {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    background-color: var(--white);
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+  }
+
+  &-image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &-title {
+    color: var(--main-accent);
+    font-size: 12px;
+
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 4px;
+  }
+
+  &-code {
+    color: var(--secondary);
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &-rate-quantity {
+    display: flex;
+    height: 16px;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  &-rate-quantity-text {
+    color: var(--main-accent);
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px;
+  }
```

No need to duplicate name classes
```suggestion
  &-rate-quantity {
    display: flex;
    height: 16px;
    flex-direction: column;
    justify-content: center;
    
    &-text {}
  }
```